sql/catalyst/src/main/java/org/apache/spark/sql/vectorized/ColumnarBatch.java

```diff
@@ -32,11 +28,11 @@
  */
 @Evolving
 public class ColumnarBatch implements AutoCloseable {
```

Comment:
I assume we want DeveloperAPI tag

it would be nice to add more to description stating why public and not final and intent on others using.

Reply:
Make sense to me. Made the change in the new commit.